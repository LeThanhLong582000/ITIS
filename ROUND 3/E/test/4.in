61
1 11
1 1
21 76
51 11
1 21
1 11
46 56
49 77
49 20
43 37
37 45
21 21
17 61
65 73
81 1
11 61
76 25
35 71
13 63
49 37
37 1
60 9
17 61
15 31
89 61
17 41
53 25
41 51
65 23
22 6
1 49
1 53
61 61
47 67
21 73
31 77
73 93
81 21
17 50
11 16
65 95
81 58
14 89
89 93
71 65
81 69
1 89
1 58
21 21
1 69
1 13
65 65
41 97
69 45
5 76
1 13
31 55
60 61
71 1
9 1
41 9
