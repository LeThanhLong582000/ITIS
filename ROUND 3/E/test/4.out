110
0
2534000
310400
420
110
6613800
14203068
952392
2517906
2758208
191000
1066956
22473960
6480
445050
3594096
6154116
664286
3271668
1332
287552
1066956
212576
29426964
480216
1744620
4354550
2222610
16556
2352
2756
13813800
9889212
2337404
5677566
46031396
2879300
715638
29700
38076954
22030574
1542522
68436404
21258306
31188404
7832
3306
191000
4692
156
17814144
15782376
9614432
141936
156
2892696
13364096
4970
72
133424
